golang/networks/network_context.go

```diff
@@ -265,3 +265,24 @@ func (networkCtx *NetworkContext) RepartitionNetwork(
 	}
 	return nil
 }
+
+func (networkCtx *NetworkContext) CheckServiceAvailability(serviceId services.ServiceID, port string, path string, initialDelaySeconds int, retries int, retriesDelayMilliseconds int, bodyText string) error {
```

Comment:
Can we add the same `Docs available at....` comment that exists on all the other funcs?

Reply:
Let's make this name match the `WaitForEndpointAvailability`